docs/reference/pip_install.rst

```diff
@@ -128,6 +128,49 @@ You can also refer to constraints files, like this::
 
     -c some_constraints.txt
 
+.. _`Example Requirements File`:
+
+Example Requirements File
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Use ``pip install -r example-requirements.txt`` to install::
+
+    #
+    ####### example-requirements.txt #######
+    #
+    ###### Requirements without Version Specifiers ######
+    nose
+    nose-cov
+    beautifulsoup4
+    #
+    ###### Requirements with Version Specifiers ######
+    #   See https://www.python.org/dev/peps/pep-0440/#version-specifiers
+    docopt == 0.6.1             # Version Matching. Must be version 0.6.1
+    keyring >= 4.1.1            # Minimum version 4.1.1
+    coverage != 3.5             # Version Exclusion. Anything except version 3.5
+    Mopidy-Dirble ~= 1.1        # Compatible release. Same as >= 1.1, == 1.1.*
+    #
+    ###### Allowing External and Unverified Packages ######
+    #   Must explicitly tell pip which packages are external and unverified.
+    #   Command-line equivalent: 'pip install pyPdf --allow-external pyPdf --allow-unverified pyPdf'
```

Comment:
Since `--allow-unverified` will be a no-op in pip 8.0, I'm not sure we actually need them in the example file ?
(same thing goes for `--allow-external`)